styles/variables-base.less

```diff
@@ -183,7 +183,8 @@
 @agate-header-title-margin: 0;
 @agate-header-title-font-size: inherit;
 @agate-header-title-text-indent: inherit;
-@agate-header-subtitle-font-size: 27px;
+@agate-header-subtitle-font-size: 24px;
+@agate-header-subtitle-padding: 9px;
```

Comment:
The padding here in the base variables file (which applies to every skin) should be set to 0, then set to 9px in silicon and gallium variables files.